### PR TITLE
refactor: remove pendingFlow logic

### PR DESCRIPTION
**Pull request by Justkant**
### 📝 Description

Remove pendingFlow logic
Use wallet-connect web3wallet existing logic for pending requests
Show pending proposals
Removed Tabs and added connect route
Update routing to be able to go back to existing proposal
React to expired proposal
Fix react-transition-group multiple warnings
Update local manifest name
Update dependencies

### ❓ Context

- **Linked resource(s)**: [] 

### 🚀 Expectations to reach

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-9879]: https://ledgerhq.atlassian.net/browse/LIVE-9879?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/screens/Connect.tsx [hunk 1]
```diff
@@ -53,67 +64,7 @@ export function Connect({ mode, initialURI }: Props) {
     navigator.userAgent?.includes("; wv") &&
```
Comment (Wozacosta) [inline src/components/screens/Connect.tsx:64]: we could be use `useMemo` here

Comment (Justkant) [reply src/components/screens/Connect.tsx:64]: correct or we can just put that outside the component actually

File: src/hooks/useWalletConnect.ts [hunk 2]
```diff
@@ -166,19 +79,40 @@ export default function useWalletConnect() {
...
+  const proposalParams = useParams({ from: "/proposal/$id" });
+
+  const onProposalExpire = useCallback(
+    (proposalExpire: Web3WalletTypes.ProposalExpire) => {
+      void queryClient
+        .invalidateQueries({
+          queryKey: pendingSessionsProposalsQueryKey,
+        })
+        .then(() => {
+          // TODO maybe remove and instead find proposal in the array from usePendingSessionsProposals on proposal route
```
Comment (Wozacosta) [inline src/hooks/useWalletConnect.ts:91]: 👍 good idea

**Review by Wozacosta — APPROVED** [review]
Let's go!

File: src/hooks/useWalletConnect.ts (hunk 2 above)
Comment (Justkant) [reply src/hooks/useWalletConnect.ts:91]: It's what is done for the session detail page, but I first tried to use the store of wallet connect directly to do the get proposal in the proposal route
In the end it feels simpler to not use the get from wallet-connect as it's easier to invalidate cache with react-query all around the app

**Pull request merged**